catapult/tickets.py

```diff
@@ -0,0 +1,94 @@
+"""
+Commands to interact with issue trackers
+"""
+from typing import List, Mapping
+
+import invoke
+
+from catapult import utils
+from catapult.integrations.git import PullRequestState
+from catapult.integrations.tracker import get_tracker
+from catapult.projects import format_projects, list_projects
```

Comment:
I'm getting an import error for `format_projects` in this line when running `catapult tickets`

Reply:
This is because this PR depends on some changes in #31. If you merge with that it should run ok.

Reply:
(Sorry it was hard to pull them apart as I'd been working on them both concurrently)